src/DotNet.Status.Web/Controllers/TelemetryController.cs

```diff
@@ -0,0 +1,78 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Kusto;
+using Kusto.Ingest;
+using System.Collections.Generic;
+
+namespace DotNet.Status.Web.Controllers
+{
+    [Route("api/telemetry")]
+    [ApiController]
+    public class TelemetryController : ControllerBase
+    {
+        private readonly ILogger<TelemetryController> _logger;
+        private readonly IOptionsSnapshot<TelemetryOptions> _options;
+        private readonly IKustoIngestClient _client;
+
+        public TelemetryController(
+            ILogger<TelemetryController> logger,
+            IOptionsSnapshot<TelemetryOptions> options,
+            IKustoIngestClient client = null)
+        {
+            _logger = logger;
+            _options = options;
+            _client = client;
+        }
+
+        [HttpPost("collect/ArcadeValidation")]
+        public async Task<IActionResult> CollectArcadeValidation([Required] ArcadeValidationData data)
+        {
+            _logger.LogInformation("Start Collect Arcade Validation data");
+
+            TelemetryOptions options = _options.Value;
+
+            if(null == options)
+            {
+                _logger.LogError("TelemetryOptions were not loaded.");
```

Comment:
This is actually impossible. The options infrastructure creates the object, its guaranteed to be not null. Its properties may not be set though.